inference/inference_benchmark/cc/Paddle/bin/run_models_benchmark.sh

```diff
@@ -11,12 +11,18 @@ export CASE_ROOT=$ROOT/bin
 export LOG_ROOT=$ROOT/log
 export gpu_type=`nvidia-smi -q | grep "Product Name" | head -n 1 | awk '{print $NF}'`
 
+# test model type
 model_type="static"
 if [ $# -ge 1 ]; then
     model_type=$1
 fi
 export MODEL_TYPE=${model_type}
 
+# test run-time device
+device_type="gpu"
+if [ $# -ge 2 ]; then
+    device_type=$2
+fi
```

Comment:
```
wget --no-proxy -q https://sys-p0.bj.bcebos.com/Paddle-UnitTest-Model/PaddleClas.tgz --no-check-certificate	    
tar -zxf PaddleClas.tgz	
```
This `PaddleClas.tgz` does not contain `AlexNet` and `yolov3_darknet`, it only contains `GhostNet_x1_3`  `Res2Net50_26w_4s`. Is it enough for us to test existing two models ?

Reply:
Did you fully download `PaddleClas.tgz`? It should contain directory tree as follow. 
```shell
.
├── infer_dygraph
│   ├── AlexNet
│   ├── DarkNet53
│   ├── DenseNet121
│   ├── DPN68
│   ├── EfficientNetB0
│   ├── GhostNet_x1_3
│   ├── GoogLeNet
│   ├── HRNet_W18_C
│   ├── InceptionV4
│   ├── MobileNetV1
│   ├── MobileNetV2
│   ├── MobileNetV3_large_x1_0
│   ├── RegNetX_4GF
│   ├── Res2Net50_26w_4s
│   ├── ResNeSt50_fast_1s1x64d
│   ├── ResNet50
│   ├── ResNet50_vd
│   ├── SE_ResNeXt50_vd_32x4d
│   ├── ShuffleNetV2_x1_0
│   ├── SqueezeNet1_0
│   ├── VGG11
│   ├── VGG16
│   └── Xception41
└── infer_static
    ├── AlexNet
    ├── DarkNet53
    ├── DenseNet121
    ├── DPN68
    ├── EfficientNetB0
    ├── GhostNet_x1_3
    ├── GoogLeNet
    ├── HRNet_W18_C
    ├── InceptionV4
    ├── MobileNetV1
    ├── MobileNetV2
    ├── MobileNetV3_large_x1_0
    ├── RegNetX_4GF
    ├── Res2Net50_26w_4s
    ├── ResNeSt50_fast_1s1x64d
    ├── ResNet50
    ├── ResNet50_vd
    ├── SE_ResNeXt50_vd_32x4d
    ├── ShuffleNetV2
    ├── SqueezeNet1_0
    ├── VGG11
    └── Xception41
```

If you test models which params size less than 100 MB would be fine
```
234M	AlexNet
160M	DarkNet53 # no need to test
32M	DenseNet121
49M	DPN68
21M	EfficientNetB0
29M	GhostNet_x1_3
27M	GoogLeNet
85M	HRNet_W18_C
165M	InceptionV4 # no need to test
17M	MobileNetV1
14M	MobileNetV2
22M	MobileNetV3_large_x1_0
86M	RegNetX_4GF
99M	Res2Net50_26w_4s
101M	ResNeSt50_fast_1s1x64d # no need to test
99M	ResNet50
99M	ResNet50_vd
107M	SE_ResNeXt50_vd_32x4d # no need to test
9.3M	ShuffleNetV2
5.0M	SqueezeNet1_0
507M	VGG11   # no need to test
89M	Xception41
```

Reply:
Hi, @OliverLPH  Thank you, I donwloaded again, and the models are exactly as listed, here. 
now LGTM. Thanks

Reply:
Brilliant! 
thanks, I will merge this PR